mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -305,11 +309,24 @@ def setup_and_check(self):
                 # the old name
                 self.includeBolus = self.namelist.getbool(
                     'config_use_standardgm')
+            try:
+                self.includeSubmesoscale = \
+                    self.namelist.getbool('config_submesoscale_enable')
+            except KeyError:
+                # an old run without submesoscale
+                self.includeSubmesoscale = False
+
             if self.includeBolus:
                 variableList.extend(
                     ['timeMonthly_avg_normalGMBolusVelocity',
                      'timeMonthly_avg_vertGMBolusVelocityTop'])
 
+            if self.includeSubmesoscale:
+                variableList.extend(
+                    ['timeMonthly_avg_normalMLEvelocity',
+                     'timeMonthly_avg_vertMLEVelocityTop'])
```

Comment:
```suggestion
                     'timeMonthly_avg_vertMLEBolusVelocityTop'])
```
I had initially understood this to be a temporary change but now see that it is, in fact, the name of the variable in MPAS-Ocean.